src/main/java/com/amazon/ion/impl/IonReaderTextRawTokensX.java

```diff
@@ -633,6 +646,18 @@ public final int nextToken() throws IOException
         case ']':
             return next_token_finish(IonTokenConstsX.TOKEN_CLOSE_SQUARE, false);
         case '(':
+            if (minorVersion > 0) {
+                c2 = read_char();
+                if (c2 == ':') {
+                    c2 = read_char();
+                    if (c2 == ':') {
+                        return next_token_finish(IonTokenConstsX.TOKEN_OPEN_PAREN_DOUBLE_COLON, true);
+                    }
+                    unread_char(c2);
+                    return next_token_finish(IonTokenConstsX.TOKEN_OPEN_PAREN_COLON, true);
+                }
+                unread_char(c2);
+            }
```

Comment:
This allows us to branch directly from these tokens to the associated expression parsing actions in Ion 1.1. See `IonReaderTextRawX`.